integration/test/ParseLocalDatastoreTest.js

```diff
@@ -2748,7 +2748,7 @@ function runTest(controller) {
       }
     });
 
-    it(`${controller.name} can select specific keys`, async () => {
+    xit(`${controller.name} can select specific keys`, async () => {
```

Comment:
Is this intentional?

Reply:
@dblythy Do you mind if I work on your branch? I see you have a few ES Module issues.